portfolio/test/main/webapp/script.spec.js

```diff
@@ -0,0 +1,38 @@
+describe('Slide Pictures', () => {
+  
+  let items = []
+  
+  beforeEach(() => {
+    items = [0, 1, 2, 3];
```

Comment:
items is not being changed, why do you need to reinitialize it every time?

Reply:
Forgot to change this back. Done